pkg/provider/acceptance_user_test.go

```diff
@@ -27,11 +27,11 @@ func TestAccUser_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("materialize_user.example_user", "email", email),
 					resource.TestCheckResourceAttr("materialize_user.example_user", "send_activation_email", "false"),
 					resource.TestCheckResourceAttr("materialize_user.example_user", "roles.0", "Member"),
-					resource.TestCheckResourceAttr("materialize_user.example_user", "verified", "false"),
+					resource.TestCheckResourceAttr("materialize_user.example_user", "verified", "true"),
 					// Data source tests
 					resource.TestCheckResourceAttrPair("data.materialize_user.user_data", "id", "materialize_user.example_user", "id"),
 					resource.TestCheckResourceAttr("data.materialize_user.user_data", "email", email),
-					resource.TestCheckResourceAttr("data.materialize_user.user_data", "verified", "false"),
+					resource.TestCheckResourceAttr("data.materialize_user.user_data", "verified", "true"),
```

Comment:
Is this just a change in the mock behavior?

Reply:
Yep, we decided to default to `true`:

https://github.com/MaterializeInc/materialize/blob/5ea2a212000a9d28dbe758879aa23c7fcfe44a20/src/frontegg-mock/src/lib.rs#L754